src/odbc/src/statement.cpp

```diff
@@ -455,7 +479,9 @@ SqlResult::Type Statement::InternalGetAttribute(int attr, void* buf, SQLINTEGER,
     case SQL_ATTR_APP_ROW_DESC: {
       SQLPOINTER* val = reinterpret_cast< SQLPOINTER* >(buf);
 
-      *val = static_cast< SQLPOINTER >(ard);
+      if (val) {
```

Comment:
On above line 472 there is a check for buf nullptr. So no need to check here.

Reply:
Removed.